packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -169,6 +179,74 @@ describe('TabsOverflow', () => {
         expect(finalLeft).to.be.lessThanOrEqual(initialLeft);
     });
 
+    xit('should scroll up to the last item and back in LTR', async () => {
+        // setUserAgent is not currently supported by Playwright
+        await setUserAgent(
+            'Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148'
+        );
+
+        const el = await renderTabsOverflow({
+            count: 8,
+            size: ElementSizes.L,
+            includeTabPanel: true,
+            dir: 'ltr',
+        });
+        await elementUpdated(el);
+        await setViewport({ width: 360, height: 640 });
+        await nextFrame();
+
+        const tabsOverflow = el.querySelector(
+            'sp-tabs-overflow'
+        ) as TabsOverflow;
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.false;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.true;
+
+        await scrollToEnd(el, RIGHT_BUTTON_SELECTOR, 'ltr');
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.true;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.false;
+
+        await scrollToEnd(el, LEFT_BUTTON_SELECTOR, 'ltr');
+
+        expect(tabsOverflow['overflowState'].canScrollLeft).to.be.false;
+        expect(tabsOverflow['overflowState'].canScrollRight).to.be.true;
+    });
+
+    xit('should scroll up to the last item and back in RTL', async () => {
```

Comment:
Would it work and make sense to let these tests run, even without setting the user agent?

Reply:
We can run the tests if we remove the `setUserAgent` part, this is how I created them in the first place. I think it would make sense to run these for desktop as well, to cover the scenario.

The thing with this approach is that they wouldn't fail even if this bug is reproducible on iOS and I think is good to signal in some way that these tests must be enabled once we have a way to run them on iPad/iOS/Android. I'm thinking to enable them in the current tests contexts and commenting the `setUserAgent` part, along with a comment like this:
```ts
// TODO: run on iPhone as per https://github.com/adobe/spectrum-web-components/pull/4722
````
What do you think, do you have any suggestions on this?

Reply:
Yes, I think that approach is good. While this bug is iOS-only, this test is running in a mobile viewport which still helps with our testing coverage 👍